### ISSUE TITLE
Clarify GHE version compatibility

### ISSUE THREAD

**Issue by AntonioL** [microsoft/vscode-pull-request-github#3427]
I am getting this error here:
```
Creating comment failed: Error: GraphQL error: Field 'path' doesn't exist on type 'PullRequestReviewThread'
GraphQL error: Field 'isOutdated' doesn't exist on type 'PullRequestReviewThread'
``` 

I think we are using an old version of GHE. Can you clarify what is the minimum version of GHE compatible with the extension?

**Comment by AntonioL** [comment]
Any update?

**Comment by vohoanglong0107** [comment]
According to this comment https://github.com/microsoft/vscode/pull/115940#issuecomment-854604614, maybe GHE versions released after 2020-09-23 or after 2.20.0 is the minimum ?

**Comment by AntonioL** [comment]
> According to this comment [microsoft/vscode#115940 (comment)](https://github.com/microsoft/vscode/pull/115940#issuecomment-854604614), maybe GHE versions released after 2020-09-23 or after 2.20.0 is the minimum ?

I can attest it is definitely not working with  an ` 2.22.0` installation.

**Comment by AntonioL** [comment]
@alexr00 any updates, did you have a chance to give a look with the relevant teams?

**Comment by alexr00** [comment]
@AntonioL GitHub Enterprise support is completely community maintained, so I don't have any insight into this question. If that changes and I start maintaining the Enterprise support too I will re-visit all Enterprise issues.

**Comment by alexr00** [comment]
I'm revisiting all GitHub Enterprise issues. I've tested all [currently supported](https://docs.github.com/en/enterprise-server@3.2/admin/all-releases) minor versions of GitHub Enterprise with the extension. I've made some fixes and in the next release we plan to announce that the extension works with GitHub Enterprise starting with GitHub Enterprise version 3.2.

@AntonioL are you still on version 2.2?
